.travis.yml

```diff
@@ -46,7 +46,7 @@ jobs:
     - stage: "Build the site for SIGMA.2"
```

Comment:
Please insert a "stage" for NIST CTCMS around here, so that the site gets test-built.

Reply:
Resolved by 24092316c33b07b791b7bf48fcb45f858f7ac451